tests/phpunit/includes/install.php

```diff
@@ -42,6 +42,15 @@
 require_once ABSPATH . 'wp-admin/includes/upgrade.php';
 require_once ABSPATH . 'wp-includes/class-wpdb.php';
 
+if ( $wpdb->last_error ) {
+	printf(
+		'Error: exiting unit tests because of the following database error: %s.' . PHP_EOL,
+		$wpdb->last_error
+	);
+	echo 'Exiting unit tests because of a database error.' . PHP_EOL,
```

Comment:
I think we can get by with just one line output. These two messages essentially say the same thing. Maybe just use the first one.

Reply:
Again, copied from the above style. I am guessing there is a reason for two messages.